extension/page-detect.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+window.pageDetect = (() => {
+	const isGistCheck = location.hostname === 'gist.github.com';
+
+	const isGist = () => isGistCheck;
+
+	const isDashboard = () => location.pathname === '/' || /^(\/orgs\/[^\/]+)?\/dashboard/.test(location.pathname);
+
+	const isRepo = () => !isGist() && /^\/[^/]+\/[^/]+/.test(location.pathname);
```

Comment:
@sindresorhus What do you mean? Just dropping the first two segments? I guess it will work, but only because we already do the `isRepo()` check first in `content.js`. Don't exclude the possibility of usernames and repositories with names such as `tree`, `blame`, `pull`, `pulls`, `issues`, `commit` and `commits`.